src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -36,47 +34,55 @@ export default {
 
 type Args = React.ComponentProps<typeof Checkbox>;
 
-/**
- * Controlled example to make checked stories interactive.
- */
-function CheckboxExample(args: Args) {
-  const [checked, setChecked] = React.useState<
-    boolean | 'indeterminate' | undefined
-  >(args.checked);
-  const handleChange = () => {
-    setChecked(!checked);
-  };
-
-  return <Checkbox checked={checked} onChange={handleChange} {...args} />;
-}
-
-export const Default: StoryObj<Args> = {
-  render: (args) => <CheckboxExample {...args} />,
-};
+export const Default: StoryObj<Args> = {};
 
 export const Checked: StoryObj<Args> = {
-  render: (args) => <CheckboxExample {...args} checked />,
+  ...Default,
+  args: {
+    defaultChecked: true,
```

Comment:
`defaultChecked` makes it start off checked, but is still uncontrolled/interactive.